src/Bridge/Symfony/Routing/Router.php

```diff
@@ -30,11 +30,20 @@
         UrlGeneratorInterface::NET_PATH => RouterInterface::NETWORK_PATH,
     ];
 
+    const CONFIG_MAP = [
+        'ABS_URL' => UrlGeneratorInterface::ABS_URL,
+        'ABS_PATH' => UrlGeneratorInterface::ABS_PATH,
+        'REL_PATH' => UrlGeneratorInterface::REL_PATH,
+        'NET_PATH' => UrlGeneratorInterface::NET_PATH,
+    ];
+
     private $router;
+    private $defaultReferenceType;
 
-    public function __construct(RouterInterface $router)
+    public function __construct(RouterInterface $router, string $defaultReferenceType)
```

Comment:
Ok I got why you've done it like that but I'm not convinced that it's a good strategy.

IMO we should not modify the router, but instead we should inject the default value in every classes needing it. This way it remains easy to change a specific value (using a compiler pass for instance) and it doesn't require this magic `::CONFIG` constant that create a coupling between components and some sort of global context we don't want. WDYT?